core/src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -278,6 +281,38 @@ public void lockTime(final MatchFinishEvent event) {
     event.getMatch().getWorld().setGameRuleValue(DO_DAYLIGHT_CYCLE, Boolean.toString(false));
   }
 
+  @EventHandler
+  public void setTime(final MatchLoadEvent event) {
+    Long time = event.getMatch().getModule(TimeSetModule.class).getTime();
+    if (time != null) {
+      event.getMatch().getWorld().setTime(time);
+    }
+  }
+
+  @EventHandler
+  public void randomTime(final MatchLoadEvent event) {
+    if (event.getMatch().getModule(TimeRandomModule.class).isTimeRandom()) {
+      /*
+      0
+      1000  /time set day
+      6000  noon, sun is at its peak
+      12000 /time set sunset
+      12610 dusk
+      13000 /time set night
+      13188 first tick when monsters spawn
+      14000
+      18000 midnight, moon is at its peak
+      21000
+      23000 /time set sunrise
+      23460 undead mobs start to burn
```

Comment:
Too arbitrary, many of these times are at night, there's 3 times where it's bright and 9 where it's dark, this would arbitrarily make most maps using random times happen during night time